pdf/core/security/auth.go

```diff
@@ -0,0 +1,9 @@
+package security
```

Comment:
https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#8-license-header-in-each-go-file